lib/src/main/java/com/orange/ods/compose/component/tab/OdsTabRow.kt

```diff
@@ -79,10 +88,10 @@ private fun PreviewOdsTabRow(@PreviewParameter(OdsTabRowPreviewParameterProvider
     with(parameter) {
         OdsTabRow(
             selectedTabIndex = selectedTabIndex,
-            tabIconPosition = if (hasLeadingIconTabs) OdsTabRowTabIcon.Position.Leading else OdsTabRowTabIcon.Position.Top,
+            tabIconPosition = if (hasLeadingIconTabs) OdsTabRow.Tab.Icon.Position.Leading else OdsTabRow.Tab.Icon.Position.Top,
```

Comment:
Depending on what we do for `OdsHorizontalCard.Image.Position` (see other comment), we should have a consistent hierarchy for both enums.

Reply:
For `OdsHorizontalCard`, I remove the `Image` level to write `OdsHorizontalCard.ImagePosition` cause `Image` is not used. Here we use `OdsTabRowTab.Icon`, do you think I should create an `OdsTabRowTab.IconPosition`?
Maybe I would have to see `Image` level as object for `OdsHorizontalCard`, I don't know...